### DISCUSSION TITLE
CloudCollab

### DISCUSSION THREAD

**Discussion by friendlyBrandon** [engageub/InternetIncome#61] [Ideas]
Hey, could you please add cloudcollab to the program?
Thank you!
Docker image: https://hub.docker.com/r/cloudcollabapp/peer
Official website: https://cloudcollab.uk/

**Comment by engageub** [comment]
Thank you for providing the website. This will be soon updated after verification. 
Thank you

**Reply by friendlyBrandon** [reply]
You could contact me/them at contact@cloudcollab.uk

**Reply by engageub** [reply]
Sure. Thank you. Changes have been pushed to test branch. Links will be added to main branch after the withdrawal is successful. 

https://github.com/engageub/InternetIncome/tree/test

Thank you

**Reply by friendlyBrandon** [reply]
Currently 0.2$

**Reply by friendlyBrandon** [reply]
Discord is friendlyBrandon
